apps/full-site-editing/full-site-editing-plugin/wpcom-block-editor-nux/src/disable-core-nux.js

```diff
@@ -0,0 +1,15 @@
+/* eslint-disable import/no-extraneous-dependencies */
+/**
+ * External dependencies
+ */
+import { select, dispatch, subscribe } from '@wordpress/data';
+import '@wordpress/nux'; //ensure nux store loads
+
+dispatch( 'core/nux' ).disableTips();
```

Comment:
Is this dispatch necessary given what is below in `subscribe` ?

Reply:
I thought it wasn't necessary too, but it turns out it is! 😄 

This ensures that the Core NUX doesn't open on editor load, even if it's supposed to.
Removing it would mean that, if the Core NUX tries to open on editor load, we instead open the WPCOM NUX **regardless of the stored user preference**.

Basically the reason for that subscribe is so that if the user manually triggers the Core NUX (AFAIK the only way is with the Welcome Guide item in the More menu), we incept, disable it and open the WPCOM NUX instead, by also setting the user attribute as "is_nux_enabled = true" in the DB.

On editor load, this is not necessarily what we want.
E.g.
- User dismissed the WPCOM NUX on Site A. The cross-site user preference is now: `is_nux_enabled = false`.
- User creates Site B.
- On first editor load, the Core NUX tries to open (as we aren't disabling it beforehand anymore).
- Our inception intervenes disabling the Core NUX, and force-enabling the WPCOM NUX instead, even though the user already dismissed it on Site A (and the user preference is `is_nux_enabled = false`).
- Additionally, the inception also force-updates the user preference as `is_nux_enabled = true` (until the WPCOM NUX is dismissed again).

It's super hard to follow, I know!
Let me know if you need me to clear up this comment a little bit 😄